serverlessworkflow/modules/ROOT/pages/eventing/handling-events-on-workflows.adoc

```diff
@@ -1,27 +1,22 @@
-= Handling events on workflows
+= Event state in Serverless Workflow
 
 :compat-mode!:
 // Metadata:
 :description: Handling events on workflows
 :keywords: kogito, workflow, serverless, event
 
-This document describes Serverless Workflow link:{spec_doc_url}#Event-State[Event state]. Event state can be used to start a new workflow instance upon the reception of a designated link:{cloud_events_url}[CloudEvent] or to pause the execution of an existing workflow instance till a designated CloudEvent is received. 
+This document describes link:{spec_doc_url}#Event-State[event state] and related examples in Serverless Workflow. You can use the event state to start a new workflow instance upon the reception of a designated link:{cloud_events_url}[CloudEvent] or to pause the execution of an existing workflow instance until a designated CloudEvent is received.
```

Comment:
"and related examples in Serverless Workflow" should be removed. There is no link to examples in this guide

Reply:
Done. Changed the sentence as follows:

This document describes link:{spec_doc_url}#Event-State[event state] in Serverless Workflow.